frontend/src/pages/profile/profile.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button } from '~/libs/components/components.js';
+import { useAppSelector } from '~/libs/hooks/hooks.js';
+
+import avatar from '../../assets/img/avatar.jpg';
+import styles from './styles.module.scss';
+
+const Profile: React.FC = () => {
+  const { user } = useAppSelector(({ auth }) => ({
+    user: auth.user,
+  }));
+
+  return (
+    <div className={styles.profile}>
+      <div className={styles.user_info_block}>
+        <div className={styles.user_avatar}>
+          <img className={styles.user_avatar_image} src={avatar} alt="avatar" />
+        </div>
+        <div className={styles.user_info}>
```

Comment:
I think styles, should be named in camel case
```suggestion
        <div className={styles.userInfo}>
```